### DISCUSSION TITLE
Where can I find your Default Theme css ?

### DISCUSSION THREAD

**Discussion by tianfanghan** [marp-team/marp#127] [Q&A] [answered]
I want to modify a new theme from importing Default theme, and I made it.
But I can't make a Invert dark class as you did in Default theme, so how can I make a Invert-able theme? 
By the way if it's allowed may I get the code of Default theme?

Thank you very much for your hard works and pretty code.

**Comment by yhatt** [comment, marked as answer]
An original source code of default theme is located in https://github.com/marp-team/marp-core/blob/main/themes/default.scss (Required [Sass](https://sass-lang.com/) to compile into CSS).

To make `invert` color scheme, you should apply `invert` class to `section` element. `section` represents each slide pages: https://marpit.marp.app/theme-css?id=html-structure

**Reply by tianfanghan** [reply]
Thanks man.
